NebulaModel/Packets/Logistics/ILSgStationPoolSync.cs

```diff
@@ -0,0 +1,129 @@
+        public Float3[] DockPos { get; set; }
+        public Float4[] DockRot { get; set; }
+        public int[] planetId { get; set; }
+        public int[] workShipCount { get; set; }
+        public int[] idleShipCount { get; set; }
+        public ulong[] workShipIndices { get; set; }
+        public ulong[] idleShipIndices { get; set; }
+        public int[] shipStationGId { get; set; }
+        public int[] shipStage { get; set; }
+        public int[] shipDirection { get; set; }
+        public float[] shipWarpState { get; set; }
+        public int[] shipWarperCnt { get; set; }
+        public int[] shipItemID { get; set; }
+        public int[] shipItemCount { get; set; }
+        public int[] shipPlanetA { get; set; }
+        public int[] shipPlanetB { get; set; }
+        public int[] shipOtherGId { get; set; }
+        public float[] shipT { get; set; }
+        public int[] shipIndex { get; set; }
+        public Double3[] shipPos { get; set; }
+        public Float4[] shipRot { get; set; }
+        public Float3[] shipVel { get; set; }
+        public float[] shipSpeed { get; set; }
+        public Float3[] shipAngularVel { get; set; }
+        public Double3[] shipPPosTemp { get; set; }
+        public Float4[] shipPRotTemp { get; set; }
+
+        public ILSgStationPoolSync() { }
+        public ILSgStationPoolSync(int[] stationGId,
+                                    int[] stationId,
+                                    Float3[] DockPos,
+                                    Float4[] DockRot,
+                                    int[] planetId,
+                                    int[] workShipCount,
+                                    int[] idleShipCount,
+                                    ulong[] workShipIndices,
+                                    ulong[] idleShipIndices,
+                                    int[] shipStationGId,
+                                    int[] shipStage,
+                                    int[] shipDirection,
+                                    float[] shipWarpState,
+                                    int[] shipWarperCnt,
+                                    int[] shipItemID,
+                                    int[] shipItemCount,
+                                    int[] shipPlanetA,
+                                    int[] shipPlanetB,
+                                    int[] shipOtherGId,
+                                    float[] shipT,
+                                    int[] shipIndex,
+                                    Double3[] shipPos,
+                                    Float4[] shipRot,
+                                    Float3[] shipVel,
+                                    float[] shipSpeed,
+                                    Float3[] shipAngularVel,
+                                    Double3[] shipPPosTemp,
+                                    Float4[] shipPRotTemp)
+        {
+            this.stationGId = new int[stationGId.Length];
```

Comment:
These could be `Array.Clone()`. Something like `this.stationGId = (int[])stationGId.Clone()`.

Reply:
implemented like that in next commit.